### DISCUSSION TITLE
Newer updates available?

### DISCUSSION THREAD

**Discussion by Timobile** [WildGums/Orchestra#953] [Q&A] [answered]
Hi,
I use your frameworks (catel, orchestra) for some years now, lately I was somewhat absent from .Net development but now I would like to create new projects using orchestra and catel.
But I am really struggling to get my project up and running. I used to use the VS templates but these include some old versions of orchestra/catel, Catel 5.11)and updating to new ones did not work for me. Also I would like to use newer .Net versions (8), that seems to be possible but I did not manage to get it to work.
Is there a newer version of the VS project templates? How can I bring up a WPF project without these templates? I am really puzzled atm. Would be nice to get some answers or links, propably I just did not find the  correct places on the internet.

How do you start a new project using .Net8, catel, orchestra?

Thanks,
Timo

**Comment by GeertvanHorrik** [comment, marked as answer]
I recommend cloning / copying one of the example apps from Orchestra. I don't think we have migrated the templates to the new project SDK yet. To be honest I don't see that happening anytime soon since we need to carefully spread our resources on the right tasks.
